Added description to help understand challenge (#34674)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/understand-bcrypt-hashes.english.md

## Description
<section id='description'>

> For the following challenges, we are going to be working on a new starter project that is different from earlier challenges. This project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-bcrypt/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-bcrypt/'>GitHub</a>.

Reviewer: ```suggestion
For the following challenges, you will be working with a new starter project that is different from earlier challenges. This project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-bcrypt/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-bcrypt/'>GitHub</a>.
```

<hr>To begin using BCrypt, add it as a dependency in your project and require it as 'bcrypt' in your server.
Submit your page when you think you've got it right.

> Remember, the tests will not pass unless you are working on the new starter project.

Reviewer: I think once at the top is enough.
```suggestion

```

## Description
<section id='description'>

> For the following challenges, you will be working with a new starter project that is different from earlier challenges. This project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-bcrypt/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-bcrypt/'>GitHub</a>.

Reviewer: @adam-weiler Quick question: Should it be `For the following challenge` or `For the following challenges`?

Author: This starter project is used for lessons 12, 13, and 14 of the unit:
-Understand BCrypt Hashes
-Hash and Compare Passwords Asynchronously
-Hash and Compare Passwords Synchronously

Lessons 1-11 of the "Information Security with HelmetJS" unit use a different starter project. That's why I thought it is important to point out the difference, so students realize to fork the new starter.

Reviewer: OK, just confirming.
